autoload/airline/extensions/ale.vim

```diff
@@ -38,12 +39,14 @@ function! airline#extensions#ale#get(type)
   endif
 
   let is_err = a:type ==# 'error'
-  let symbol = is_err ? s:error_symbol : s:warning_symbol
 
-  let is_err = a:type ==# 'error'
-  let counts = ale#statusline#Count(bufnr(''))
+  if ale#engine#IsCheckingBuffer(bufnr('')) == 1
+    return is_err ? '' : s:checking_symbol
```

Comment:
instead on relying on `s:checking_symbol`, please make this `get(g:, 'airline#extensions#chechking_symbol', '...')`

Reply:
s:checking_symbol is handled just like s:warning_symbol and s:error_symbol. The definition of s:*_symbol can be found at the top of the file and airline#extensions#ale#get refers to all of them.

Please let me know why this new option needs to be handled in a different way.